include/faunus/group.h

```diff
@@ -22,8 +22,16 @@ namespace Faunus
   {
   private:
       int molsize; // Number of atoms per molecule
+
+
   public:
       typedef PropertyBase::Tid Tid;
+      int molIndex = -1; /// \brief Index of group in molecular tracker (sum of previous molecules in vector and on lower TID)
```

Comment:
I'm not sure `group` should be responsible for knowing where it is located in `Space`. This adds an interdependency that should be avoided. Is the problem that you in energy functions get a group and not an index?